CalendarApp/NUnitTestProject/BookingServiceTests.EventManagerTest.cs

```diff
@@ -0,0 +1,542 @@
+﻿using System.Collections.Generic;
+using NUnit.Framework;
+using BookingRoom.Helpers;
+using BookingRoom.Models.GoogleEvent;
+using Google.Apis.Calendar.v3.Data;
+
+namespace NUnitTestProject
+{
+    [TestFixture]
+    public partial class BookingServiceTests
+    {
+        [Test]
+        public void EventManager_Singlet_FindConflicts_Case1()
```

Comment:
Is there resource where `Case*` are discrabed more informal or explained, otherwise I'd suggest to rename these to more clean?